scholarx/archive/2020/mentees.json

```diff
@@ -0,0 +1,426 @@
+    },
+
+    {
+        "name": "Windhula Gammanpila",
+        "image": "windhula.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Sri Jayawardenapura",
+        "linkedin": "https://www.linkedin.com/in/windhula-gammanpila-6a8264151/"
+    },
+
+    {
+        "name": "Muhammed Imdaad",
+        "image": "muhammed.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/muhammed-imdaad-23356a17b/"
+    },
+
+    {
+        "name": "Ranjula Hettiarachchi",
+        "image": "ranjula.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/ranjula-hettiarachchi/"
+    },
+
+    {
+        "name": "Dedunu Karunarathne",
+        "image": "dedunu.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/dedunu-karunarathne/"
+    },
+
+    {
+        "name": "Suchitha Dehigaspitiya",
+        "image": "suchitha.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/suchitha-dehigaspitiya/"
+    },
+
+    {
+        "name": "Sahan Samuditha",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Nishanta Rajakaruna",
+        "university": "University of Sri Jayawardenapura",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Visal Senathilaka",
+        "image": "visal.jpg",
+        "mentor": "Nishanta Rajakaruna",
+        "university": "University of Sri Jayawardenapura",
+        "linkedin": "https://www.linkedin.com/in/visal-senathilaka-959bb4145/"
+    },
+
+    {
+        "name": "John Kotelawala",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Nivanthika Wimalasena",
+        "university": "London School of Economics and Political Science",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Isuru Thilakasiri",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Nivanthika Wimalasena",
+        "university": "University of Peradeniya",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Arundika Weerasekera",
+        "image": "arundika.jpg",
+        "mentor": "Nushelle De Silva",
+        "university": "Sri Lanka Institute of Information Technology",
+        "linkedin": "https://www.linkedin.com/in/arundika-weerasekera/"
+    },
+
+    {
+        "name": "Yashodha Warakapitiya",
+        "image": "yashodha.jpg",
+        "mentor": "Nushelle De Silva",
+        "university": "General Sir John Kotelawala Defence University",
+        "linkedin": "https://www.linkedin.com/in/yashodha-warakapitiya-094a4015a/"
+    },
+
+    {
+        "name": "Amaya Dharmasiri",
+        "image": "amaya.jpg",
+        "mentor": "Rukshan Batuwita",
```

Comment:
It is Dr. Rukshan Batuwita. [Apply for all the entries and please check spellings.]